Compiler error: ‘SplitBehaviorFlags’ in ‘class QString’ does not name a type

Hello,
I can't compile MediaElch from github (2.8.1-dev). Every time i get the following error while compiling main.cpp:

`In file included from ../src/globals/Helper.h:5,
                 from ../src/data/MediaInfoFile.cpp:3:
../src/globals/Globals.h:17:36: error: ‘SplitBehaviorFlags’ in ‘class QString’ does not name a type
 using ElchSplitBehavior = QString::SplitBehaviorFlags;
                                    ^~~~~~~~~~~~~~~~~~
`

My system is Debian Buster with Qt 5.11.3

Thank you!

--- a/src/globals/Globals.h
+++ b/src/globals/Globals.h
@@ -14,7 +14,7 @@
 
 // Required for smoother upgrade to Qt 6 while still working with Qt 5
 #if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
-using ElchSplitBehavior = QString::SplitBehaviorFlags;
+using ElchSplitBehavior = QString::SplitBehavior;
 #else
 using ElchSplitBehavior = Qt::SplitBehaviorFlags;
 #endif

